chrome/content/zotero/xpcom/utilities.js

```diff
@@ -455,11 +455,15 @@ Zotero.Utilities = {
 	 *   <ZOTEROBREAK/> => <br/>
 	 *   <ZOTEROHELLIP/> => &#8230;
 	 *
-	 * @param {String} str
+	 * @param {String|Number} str
 	 * @return {String}
 	 */
 	"htmlSpecialChars":function(str) {
-		if (str !== undefined && typeof str != 'string') {
+		if ((str || str == 0) && typeof str != 'string') {
```

Comment:
Did you mean `str === 0`?

If we want to keep `true` support for now, seems like this should just be `if (typeof str == 'number' || str === true)`?